tests/scripts/test_config.sh

```diff
@@ -29,6 +30,14 @@ expect_fail pulp --username test --password test --client "/some/path" status
 expect_fail pulp --key "/some/path" file remote list
 
 
+# CONFFIG PROFILE
```

Comment:
```suggestion
# CONFIG PROFILE
```

Reply:
Hmm this keyboard of mine is really oversensitive...